website/content/en/preview/concepts/setup.md

```diff
@@ -0,0 +1,612 @@
+---
```

Comment:
Good question. I think this will be used with the getting started docs. However, this is a passive doc, meaning that it is not procedural in nature. We can put it in the new Reference section you are proposing, with a pointer from the place in the Getting Started guide that uses cloudformation.yaml.